engine/src/test/java/org/hibernate/validator/test/internal/engine/ValidatorTest.java

```diff
@@ -272,7 +272,7 @@ public String getE() {
 
 	class Ticket {
 		@NotNull
-		Float €price;
```

Comment:
Please don't remove the "€" character, it was the very basis for this particular test ;)